pyvrp/cpp/educate/LocalSearch.h

```diff
@@ -95,15 +94,19 @@ class LocalSearch
                       CostEvaluator const &costEvaluator);
 
     /**
-     * Performs a more intensive local search around the given individual,
-     * using route-based operators and subpath enumeration. Returns a new,
-     * hopefully improved individual.
+     * Performs a more intensive route-based local search around the given
+     * individual, and returns a new, hopefully improved individual.
      */
     Individual intensify(Individual &individual,
                          CostEvaluator const &costEvaluator,
                          int overlapToleranceDegrees = 0);
 
-    LocalSearch(ProblemData &data, XorShift128 &rng, Neighbours neighbours);
+    /**
+     * Shuffles the order in which the node and route pairs are evaluated.
+     */
```

Comment:
Suggestion: also add something about the operators like "and the order in which the node and route operators are applied."